exporter/opentelemetry-exporter-prometheus-remote-write/src/opentelemetry/exporter/prometheus_remote_write/__init__.py

```diff
@@ -142,50 +161,132 @@ def export(
     def shutdown(self) -> None:
         raise NotImplementedError()
 
-    def convert_to_timeseries(
+    def _convert_to_timeseries(
         self, export_records: Sequence[ExportRecord]
     ) -> Sequence[TimeSeries]:
-        raise NotImplementedError()
-
-    def convert_from_sum(self, sum_record: ExportRecord) -> TimeSeries:
-        raise NotImplementedError()
+        timeseries = []
+        for export_record in export_records:
+            aggregator_type = type(export_record.aggregator)
+            converter = self.converter_map.get(aggregator_type)
+            if converter is None:
+                raise ValueError(
```

Comment:
Do we want to raise an exception? This could happen during runtime and we have a strict "no-crashing user's app" policy.

Reply:
Would the standard then be to log a warning and then abort the export of that specific record?

Reply:
Implemented the above mentioned logging ^